nuxeo-services/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/web/common/requestcontroller/filter/NuxeoCorsFilter.java

```diff
@@ -32,15 +35,29 @@
  * Nuxeo CORS filter wrapper to com.thetransactioncompany.cors.CORSFilter allowing to configure cors filter depending of
  * the request url. Each time a request matchs a contribution is found, CORSFilter had to be re-initialized to change
  * his configurations.
- * 
+ *
  * @author <a href="mailto:ak@nuxeo.com">Arnaud Kervern</a>
  * @since 5.7.2
  */
 public class NuxeoCorsFilter extends CORSFilter {
 
+    public static final String FRAME_OPTIONS_HEADER = "X-Frame-Options";
+
+    public static final String FRAME_OPTIONS_DEFAULT = "";
+
+    /** Frame options. Can be DENY, SAMEORIGIN, ALLOW-FROM someuri, or blank. */
+    public static final String FRAME_OPTIONS_PROPERTY = "nuxeo.frame.options";
+
+    public String frameOptions = Framework.getProperty(FRAME_OPTIONS_PROPERTY, FRAME_OPTIONS_DEFAULT);
+
     @Override
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException,
             ServletException {
+        // TODO move this to a separate filter, for instance one dealing with Content-Security-Policy
```

Comment:
TODO should be associated with a JIRA issue (actually, this TODO seems useless on maintenance branch)